huggingface/tensorflow/buildspec.yml

```diff
@@ -27,7 +27,22 @@ context:
       target: config.properties
 
 images:
-  BuildHuggingFaceTensorflowCPUPTInferencePy3DockerImage:
+ BuildHuggingFaceTensorflowGpuPy37Cu110TrainingDockerImage:
```

Comment:
Looks like there are some indentation issues here. Could you make sure spacing is correct on this line?

Reply:
thanks!